.github/workflows/rpc-compat.yml

```diff
@@ -0,0 +1,34 @@
+name: rpc-compat
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  check-rpc-for-breaking-changes:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup sentry env
+        uses: ./.github/actions/setup-sentry
+        id: setup
+      - name: Dump RPC schema for new version
+        id: newschema
+        run: |
+          mkdir schemas
+          sentry rpcschema --partial > schemas/new_schema.json
+      - name: Dump RPC schema for old version
+        id: oldschema
+        run: |
+          git checkout `git merge-base origin/${{ github.base_ref }} origin/${{ github.head_ref }}`
+          sentry rpcschema --partial > schemas/old_schema.json
+      - name: Invoke oasdiff on two outputs
+        run: |
+          docker run --rm -t \
+            -v $(pwd)/schemas:/schemas:ro \
+            tufin/oasdiff breaking /schemas/old_schema.json /schemas/new_schema.json \
```

Comment:
a little nervous running some random docker image here -- can we make sure we're pinning a revision on this?

Reply:
Would it be preferable for us to make a fork/mirror of this image and pin to a sha in our copy?

Reply:
that would work yeah